v3/src/models/data/plotted-value-formula-adapter.ts

```diff
@@ -0,0 +1,146 @@
+import { makeObservable, observable } from "mobx"
+import { ICase } from "./data-set-types"
+import { formulaError } from "./formula-utils"
+import { math } from "./formula-fn-registry"
+import { IFormula } from "./formula"
+import type {
+  IFormulaAdapterApi, IFormulaContext, IFormulaExtraMetadata, IFormulaManagerAdapter
+} from "./formula-manager"
+import type { IGraphContentModel } from "../../components/graph/models/graph-content-model"
+import { onAnyAction } from "../../utilities/mst-utils"
+import {
+  isPlottedValueAdornment
+} from "../../components/graph/adornments/univariate-measures/plotted-value/plotted-value-adornment-model"
+import { IAnyStateTreeNode } from "@concord-consortium/mobx-state-tree"
+import { getFormulaManager } from "../tiles/tile-environment"
+import { FormulaMathJsScope } from "./formula-mathjs-scope"
+
+// This should match type of PlottedValueAdornmentModel. Currently all these models use regular string instead of
+// enums / constants / literals, so I can only copy the value and save it here.
```

Comment:
I should fix this. I made [a new chore story](https://www.pivotaltracker.com/story/show/186297171) for it.